inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -1248,14 +1249,17 @@ InferenceEngine::Precision MKLDNNNode::getRuntimePrecision() const {
 MKLDNNNode* MKLDNNNode::NodesFactory::create(const std::shared_ptr<ngraph::Node>& op, const mkldnn::engine& eng,
                                              const MKLDNNExtensionManager::Ptr& extMgr, MKLDNNWeightsSharing::Ptr &w_cache) {
     MKLDNNNode *newNode = nullptr;
-
+    std::string errorMessage;
     try {
         std::unique_ptr<MKLDNNNode> ol(createNodeIfRegistered(MKLDNNPlugin, Generic, op, eng, w_cache));
         if (ol != nullptr && ol->created(extMgr))
             newNode = ol.release();
     } catch (const details::InferenceEngineException& ex) {
-        if (ex.getStatus() != NOT_IMPLEMENTED)
+        if (ex.getStatus() != NOT_IMPLEMENTED) {
             throw;
```

Comment:
Minor comment
Maybe we should add some error message like: ```"MKLDNNNode creation was failed on Generic factory. Node type: " + op->get_type_name() + ". Node name: " + op->get_friendly_name()``` ?
Because it can be confusing for user if application just crash without error message.
Or are you planning to remove these exception throws at all?

Reply:
It won't crash w/o error message because throw w/o expression just propagates original exception further: https://en.cppreference.com/w/cpp/language/throw. So I just rethrow `ex` in this case.